createContainer.js

```diff
@@ -41,9 +41,9 @@ module.exports = function createContainer(Component) {
 	return React.createClass({
 		displayName: componentName + "Container",
 		statics: {
-			chargeStores: function(stores, params, callback) {
+			chargeStores: function(stores, params, query, callback) {
```

Comment:
This way it would be a breaking change. Could you add backward compatibility here? So one can call `chargeStores(stores, params, callback)` and `chargeStores(stores, params, query, callback)`.